### ISSUE TITLE
checker reports non-found index, changes and directories as "good"

### ISSUE THREAD

**Issue by bernhardreiter** [csaf-poc/csaf_distribution#160]
Testing a trusted provider with broken ROLIE feed, and no index.txt, no changes.csv and no directory listing, leads to a result with is missleading:

```json
{[
            "Loading ROLIE feed failed: parsing time \"2020-09-08T00:00Z\" as \"2006-01-02T15:04:05Z07:00\": cannot parse \"Z\" as \":\"."
          ]
        },
        {
          "num": 8,
          "description": "security.txt",
          "messages": [
            "Found good security.txt."
          ]
        },

[...]

        {
          "num": 12,
          "description": "index.txt",
          "messages": [
            "Found good index.txt."
          ]
        },
        {
          "num": 13,
          "description": "changes.csv",
          "messages": [
            "Found good changes.csv."
          ]
        },
        {
          "num": 14,
          "description": "Directory listings",
          "messages": [
            "All directory listings are valid."
          ]
        },
  "version": "v0.9.0",
  "date": "2022-06-01T17:36:42Z"
```

expected would be a warning.

**Comment by bernhardreiter** [comment]
Needs a retest, is potentially already improved.

**Comment by bernhardreiter** [comment]
See result from current test of main e.g. https://github.com/csaf-poc/csaf_distribution/actions/runs/2711754318 .
In the default configuration and in the itest, there are **no** filex index.txt or changes.cvs, still the report says:

```
    <dd>- INFO: Found good index.txt.</dd>
    <dd>- INFO: Found good changes.csv.</dd>
```

**Comment by JanHoefelmeyer** [comment]
Further test and differentiate whether index.txt or changes.csv are empty. Trying  to fix this in https://github.com/csaf-poc/csaf_distribution/tree/find-missing

**Comment by JanHoefelmeyer** [comment]
Fixed by #269
